packages/react/src/themes/teams/siteVariables.ts

```diff
@@ -17,7 +17,7 @@ export {
 // BORDER STYLES
 //
 export const borderWidth = '1px'
-export const borderRadius = '2px'
+export const borderRadius = '3px'
```

Comment:
just wanted to check and see if you meant to change this for all the instances currently using this value? should we only ever be using a 3px border radius?

Reply:
Good catch, I don't think these even affects my changes.